src/lib.rs

```diff
@@ -325,7 +325,7 @@ type ConvertResult = Result<Vec<u8>, Error>;
 ///
 /// # Panics
 /// Function will panic if attempting to convert `from` or `to` a bit size that
-/// is larger than 8 bits.
+/// is larger than 8 bits or 0 bits.
```

Comment:
Nit: I think it reads better as `is 0 bits or larger than 8 bits.`

Reply:
Yes, I thought about writing `exactly 0` and it felt a bit over the top, but just changing the order makes it quite clear.